### DISCUSSION TITLE
Demo: blocks aren't dropping into canvas for me

### DISCUSSION THREAD

**Discussion by gkustas** [GrapesJS/grapesjs#3610] [Q&A]
I'm a GrapesJS newbie, and I'm sure I'm doing something wrong...

I am following the "Getting Started" documentation from here: https://grapesjs.com/docs/getting-started.html

The main component is displaying properly, and the three blocks are also displaying below it exactly as shown in the docs

When I drag a block to the canvas, nothing is happening.  

Help is MUCH appreciated.  I tried adding my code to this thread, but it won't display. It's trying to render it as HTML, even though I'm using the "insert code" toolbar button.

**Comment by gkustas** [comment]
Additional info:

Per this discussion (https://github.com/artf/grapesjs/issues/580) I added the following code and the alert was never displayed, meaning I suppose, that the drop event doesn't occur:

	editor.on('block:drag:stop', model => console.log('dropped ', model))

ALSO: If it makes a difference, I am hosting my web page using .Net Core 5.0 using Razor pages.

_______________________________________
UPDATE:
I cloned the source code, rebuilt with yarn, and ran the test page using yarn start. It opened the page in my default browser (Chrome) and ran successfully - including the drag and drop blocks.

So, the original problem has something to do with my .Net Core configuration (or running in the visual studio debugger). Not sure what it is, but if I figure it out I'll re-post.

**Comment by ronaldohoch** [comment]
Can you post a live sample in codepen.io or stackblitz?

**Reply by gkustas** [reply]
Thanks Ronald, but I have it working now in a simple web server. The problem had to do with the .Net Core server (or Razor pages) which I don't need to use.
